### ISSUE TITLE
ls problem

### ISSUE THREAD

**Issue by TimoGo** [kellyjonbrazil/jc#79]
As far as I can see everything works fine, but --ls causes troubles:

`ls -l /usr/bin | jc --ls
`
The whole data is in "filename":
```
  {
    "filename": "lrwxrwxrwx 1 root   root            8 Jan 25  2020 zsh -> /bin/zsh"
  }
```

OS: Debian 10  
Version: 1.13.4  (binary and pip3)

Ive tried debian 9+10 containers, both worked well, maybe an environment or language problem?

**Comment by kellyjonbrazil** [comment]
Thank you for reporting this. Could you post the output of ls before it gets piped to jc? 

Also, have you tried prepending LANG=C to the command?

Thanks!

**Comment by TimoGo** [comment]
yes its a LANG problem and I found this issue with the alias stuff in the meantime:  https://github.com/kellyjonbrazil/jc/issues/27

The command works well when I switch LANG to "C".
`   LANG=C ls -l /usr/bin | jc --ls |jq .
`
But I found out, that this is only a matter of the input stream.
this works also well:
` LANG="" ls -l /usr/bin | LANG="de_DE.UTF-8" jc --ls |jq .`
So its enough to change LANG for the ls-command:


The following alias doesnt work:
```
   alias jc='LANG=C jc'
   ls -l /usr/bin | jc --ls |jq .
```
Its because the ls output of stdin is still the same.

The best workaround is to use the magic syntax combined with the alias:
```
 alias jc='LANG=C jc'
 jc ls -l /usr/bin | jq .
```
